proc.go

```diff
@@ -2,11 +2,53 @@ package main
 
 import (
 	"errors"
+	"fmt"
 	"os"
+	"os/exec"
 	"sync"
 	"time"
 )
 
+// spawnProc starts the specified proc, and returns any error from running it.
+func spawnProc(proc string, errCh chan<- error) {
+	procObj := procs[proc]
+	logger := createLogger(proc, procObj.colorIndex)
+
+	cs := append(cmdStart, procObj.cmdline)
```

Comment:
I'm not sure but `append(cmdStart, ...)` modify cmdStart. So it may break second call of spawnProc.

Reply:
I don't think so... see https://play.golang.org/p/iIez3WbTbR9

```go
package main

import "fmt"

func main() {
	a := []string{"one", "two"}
	c := append(a, "three")
	d := append(a, "four")
	fmt.Printf("%#v\n", a)
	fmt.Printf("%#v\n", c)
	fmt.Printf("%#v\n", d)
}
```

```
[]string{"one", "two"}
[]string{"one", "two", "three"}
[]string{"one", "two", "four"}
```

Reply:
Ah, sorry. cap of literal is same as length.